PowerEditor/src/WinControls/PluginsAdmin/pluginsAdmin.rc

```diff
@@ -39,7 +39,7 @@ EXSTYLE WS_EX_TOOLWINDOW
 CAPTION "Plugins Admin"
 FONT 8, "MS Shell Dlg", 0, 0, 0x0
 BEGIN
-	RTEXT           "Search: ",IDC_PLUGINADM_SEARCH_STATIC,50,38,50,8
+	LTEXT           "Search: ",IDC_PLUGINADM_SEARCH_STATIC,50,38,50,8
```

Comment:
For the sake of translation, I keep more space on the left for LTEXT.

Reply:
@donho This can be achieved by shifting edit control and 'next' button little bit instead of overlapping.